docs/src/content/docs/api/calls.mdx

```diff
@@ -410,9 +421,83 @@ Lists all messages generated during the given call.
   </TabItem>
 </Tabs>
 
+## Get Call Recording
+<Badge text="GET /calls/{uuid}/recording" variant="note" size="large" />
+
+Returns a link to the recording of the call (via a 302 redirect to the file location).
```

Comment:
What if the recording isn't ready yet?

Reply:
@mdepinet what happens if the recording isn't ready? how long does it take to generate?

Reply:
Typically if the recording isn't ready it will be because the call hasn't ended yet.  We stream out the recording, so it should be ready within milliseconds of the call ending.  But regardless, we return a 425 (Too Early) if the recording isn't ready.

Reply:
thx